### PR TITLE
Activity Page Updates

### PR DESCRIPTION
**Pull request by jakekrog**
This branch makes the following changes to the Activity page:
- Update list layout per HIG
- Give user completion message when all activities are completed for that day
- Remove color indicators from activities

### CODE REVIEW

File: mPowerSDK/Activities/APHActivitiesTintedTableViewCell.m [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	[super configureWithTaskGroup:taskGroup
+					  isTodayCell:cellRepresentsToday
+				showDebuggingInfo:shouldShowDebuggingInfo];
+
+	self.titleLabel.textColor =
+		cellRepresentsToday && !taskGroup.isFullyCompleted || taskGroup.task.taskIsOptional.boolValue ?
+			[UIColor appSecondaryColor1] : [UIColor appSecondaryColor3];
+
+	if (cellRepresentsToday) {
+		self.confirmationView.completedBackgroundColor = [UIColor colorWithRed:0.39 green:0.76 blue:0.46 alpha:1];
```
Comment (syoung-smallwisdom) [inline mPowerSDK/Activities/APHActivitiesTintedTableViewCell.m]: hardcoded color

File: mPowerSDK/Activities/APHActivitiesViewController.m [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+									 compatibleWithTraitCollection:nil];
+		headerView.titleLabel.textAlignment = NSTextAlignmentCenter;
+		headerView.subTitleLabel.textAlignment = NSTextAlignmentCenter;
+	} else {
+		headerView.backgroundImageView.image = nil;
+		headerView.titleLabel.textAlignment = NSTextAlignmentLeft;
+		headerView.subTitleLabel.textAlignment = NSTextAlignmentLeft;
+	}
+
+	if ([section isKeepGoingSection]) {
```
Comment (syoung-smallwisdom) [inline mPowerSDK/Activities/APHActivitiesViewController.m]: empty if statement

**Comment by syoung-smallwisdom** [comment]
few nits. otherwise, looks fine.

**Comment by syoung-smallwisdom** [comment]
Waiting for review and test before accepting into v1.3

File: mPowerSDK/Activities/APHActivitiesViewController.m [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+//  APHActivitiesViewController.m
+//  mPowerSDK
+//
+//  Created by Everest Liu on 2/4/16.
+//  Copyright © 2016 Thread, Inc. All rights reserved.
+//
+
+#import "APHActivitiesViewController.h"
+
+static NSString *const kAPCSectionTitleKeepGoing = @"Additional Activities";
```
Comment (Erin-Mounts) [inline mPowerSDK/Activities/APHActivitiesViewController.m]: these need to be localized strings

**Comment by jakekrog** [comment]
I noticed that the section header is being configured in a separate PR for AppCore (https://github.com/Sage-Bionetworks/AppCore/pull/84), so I removed it from this PR.

**Comment by Erin-Mounts** [comment]
Please manually merge against master and push changes.

**Comment by evliu** [comment]
Merge conflicts resolved. Requires AppCore PR https://github.com/Sage-Bionetworks/AppCore/pull/84

**Comment by syoung-smallwisdom** [comment]
closing

**Pull request closed without merging**